.hygen/generate/relational-resource/prompt.js

```diff
@@ -0,0 +1,20 @@
+module.exports = [
+  {
+    type: 'confirm',
+    name: 'isAddTestCase',
+    message: 'Do you want to add test cases and mock data?',
+    initial: true,
+  },
+  {
+    type: 'multiselect',
+    name: 'functionalities',
+    message: 'Select the functionalities you want to include:',
+    choices: [
+      { name: 'create', value: 'create' },
+      { name: 'findAll', value: 'findAll' },
+      { name: 'findOne', value: 'findOne' },
+      { name: 'update', value: 'update' },
+      { name: 'delete', value: 'delete' },
+    ],
```

Comment:
Can we have short names like instead of create user just add c.

Reply:
It is basically multi-select user don't have to write it. We can write "c"  but I think it's more ambiguous